randovania/games/am2r/json_data/Distribution Center.txt

```diff
@@ -233,18 +233,14 @@ Extra - map_name: rm_a5c05
   * Normal Door to Energy Distribution Activation Shaft/Door to Meat Boy Room
   * Extra - instance_id: 133616
   > Door to Energy Distribution Tower West
-      All of the following:
-          Can Use Any Bombs
-          Morph Ball or Screw Attack
+      Can Use Any Bombs
```

Comment:
Can you add a `TODO: combat for enemies` somewhere? Either only for one connection here, or in the room description